cypress/e2e/marc/marc-authority/reporting-marc-authority/marc-authority-reporting-not-include-data-deleted-record.cy.js

```diff
@@ -0,0 +1,206 @@
+    fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+    jobProfileToRun: 'Default - Create SRS MARC Authority',
+  },
+];
+
+const expectedJobValues = {
+  status: 'Successful',
+  jobType: 'MARC authority headings updates',
+  description: 'List of updated MARC authority (1XX) headings',
+  outputType: 'MARC authority headings updates (CSV)',
+};
+const today = DateTools.getFormattedDate({ date: new Date() }, 'MM/DD/YYYY');
+const todayWithoutPaddingZero = DateTools.clearPaddingZero(today);
+const tomorrow = DateTools.getTomorrowDayDateForFiscalYear();
+const createdAuthorityID = [];
+
+describe('MARC -> MARC Authority -> Reporting MARC authority', () => {
+  before('Creating user and uploading files', () => {
+    cy.createTempUser([
+      Permissions.uiMarcAuthoritiesAuthorityRecordView.gui,
+      Permissions.uiMarcAuthoritiesAuthorityRecordEdit.gui,
+      Permissions.uiQuickMarcQuickMarcAuthoritiesEditorAll.gui,
+      Permissions.inventoryAll.gui,
+      Permissions.uiMarcAuthoritiesAuthorityRecordDelete.gui,
+      Permissions.uiQuickMarcQuickMarcBibliographicEditorAll.gui,
+      Permissions.uiQuickMarcQuickMarcAuthorityLinkUnlink.gui,
+      Permissions.uiQuickMarcQuickMarcAuthorityLinkUnlink.gui,
+      Permissions.exportManagerAll.gui,
+    ]).then((createdUserProperties) => {
+      testData.userProperties = createdUserProperties;
+
+      marcFiles.forEach((marcFile) => {
+        cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(
+          () => {
+            DataImport.verifyUploadState();
+            DataImport.uploadFileAndRetry(marcFile.marc, marcFile.fileName);
+            JobProfiles.waitLoadingList();
+            JobProfiles.search(marcFile.jobProfileToRun);
+            JobProfiles.runImportFile();
+            JobProfiles.waitFileIsImported(marcFile.fileName);
+            Logs.checkStatusOfJobProfile('Completed');
+            Logs.openFileDetails(marcFile.fileName);
+            Logs.getCreatedItemsID().then((link) => {
+              createdAuthorityID.push(link.split('/')[5]);
+            });
+          },
+        );
+      });
+
+      cy.visit(TopMenu.inventoryPath).then(() => {
+        InventoryInstances.waitContentLoading();
+        InventoryInstances.searchByTitle(createdAuthorityID[0]);
+        InventoryInstances.selectInstance();
+        // wait for detail view to be fully loaded
+        cy.wait(1500);
+        InventoryInstance.editMarcBibliographicRecord();
+        InventoryInstance.verifyAndClickLinkIcon(testData.tag240);
+        MarcAuthorities.switchToSearch();
+        InventoryInstance.verifySelectMarcAuthorityModal();
+        InventoryInstance.searchResults(testData.marcValue);
+        MarcAuthoritiesSearch.selectAuthorityByIndex(0);
+        InventoryInstance.clickLinkButton();
+        QuickMarcEditor.pressSaveAndClose();
+        QuickMarcEditor.checkAfterSaveAndClose();
+        InventoryInstance.waitLoading();
+      });
```

Comment:
Test case requires you to create link in fileds 100 and 240 of Marc Bib file, but I see you are only linking 240 field, are you linking 100 field of marc bib with marc auth too?

Reply:
No I`ve created only for 240 field, now I`ll change it!